notebooks/curation_api/README.md

```diff
@@ -0,0 +1,24 @@
+# Discover API notebooks
+
+The Discover API notebooks are pre-baked scripts to facilitate use of the 
+[Discover API](https://api.cellxgene.cziscience.com/curation/ui/#/), which itself supports 
+curation flow interactions with the [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) tool.
+
+## Notebook usage
+
+Many notebooks have a few fields that require user input, such as the relevant Collection id or Dataset id.
+
+### Python notebooks: `python/`
+
+These notebooks are the most user-friendly; start here if you're unsure. All access token generation, url construction, and response 
+handling is abstracted away from the user. Unitary interactions such as getting a Dataset, creating a Collection, etc., 
+are each handled by single function calls.
+
+### "Raw" Python notebooks: `python_raw/`
+
+These notebooks show the construction of http requests at a more granular level, and as such may be useful for technical
+users.
+
```

Comment:
and ~as such~ may be useful for technical users.

This might be useful for someone who wants to _port_ the REST API to another language or wants more control of the responses?

Reply:
or wants more control over the _request_, yes (for us, at the moment, and partners, in the case of hitting a Discover API on rdev...)

Reply:
changed to only reference consumer actions...